physicsnemo/models/domino/model.py

```diff
@@ -146,7 +184,12 @@ def __init__(self, input_features, model_parameters, grid_resolution=[256, 96, 6
 
         self.activation = F.relu
 
-    def forward(self, x, radius=0.025, neighbors_in_radius=10):
+    def forward(
+        self, x: torch.Tensor, radius: float = 0.025, neighbors_in_radius: int = 10
+    ) -> torch.Tensor:
+        """
```

Comment:
I don't fully understand this layer yet.  The fully-connecteds make sense, but the expand and reshape I don't totally follow.  Can we add a description of what is happening?

Also, could remove `nx, ny, nz = (self.grid_resolution[0] ...` and just do
```
x = torch.resshape(x, (batch_size, x.shape[-1], *self.grid_resolution)
```